pkg/apis/core/v1beta1/types.go

```diff
@@ -152,14 +152,35 @@ type DataVolumeSourceS3 struct {
 
 // DataVolumeSourceRegistry provides the parameters to create a Data Volume from an registry source
 type DataVolumeSourceRegistry struct {
-	//URL is the url of the Docker registry source
+	//URL is the url of the registry source (starting with the scheme: docker, oci-archive, imagestream)
 	URL string `json:"url"`
+	//PullMethod can be either "pod" (default import), or "node" (node docker cache based import)
+	PullMethod RegistryPullMethod `json:"pullMethod,omitempty"`
 	//SecretRef provides the secret reference needed to access the Registry source
 	SecretRef string `json:"secretRef,omitempty"`
 	//CertConfigMap provides a reference to the Registry certs
 	CertConfigMap string `json:"certConfigMap,omitempty"`
 }
 
+const (
+	// RegistrySchemeDocker is docker scheme prefix
+	RegistrySchemeDocker = "docker"
+	// RegistrySchemeOci is oci-archive scheme prefix
+	RegistrySchemeOci = "oci-archive"
+	// RegistrySchemeImageStream is image stream scheme prefix
+	RegistrySchemeImageStream = "imagestream"
```

Comment:
I don't see any actual support for this yet in here (not adding annotations to import pod).  What is the plan?

Reply:
Yep, it should be moved to the controller PR and somehow got here. Will move.

Reply:
Given recent conversations, should `RegistrySchemeImageStream` be deleted?

Reply:
Sure, working on the IS change it this morning.

Reply:
Done